docs/stagenet.md

```diff
@@ -14,13 +14,13 @@ The atomic swap daemon requires a connection to a monero-wallet-rpc process conn
 
 For Linux 64-bit, you can do:
 ```bash
-curl -L https://downloads.getmonero.org/cli/linux64 > monero.tar.bz2
-tar xjvf monero.tar.bz2
+bash scripts/install-monero-linux.sh
```

Comment:
The script has execute permissions (which git respects/maintains) and it starts with `#!/bin/bash`. Any reason to not execute it directly?  i.e. `./scripts/install-monero-linux.sh`?

Reply:
good point, updated!